common/ts/prepare-task.ts

```diff
@@ -54,6 +54,9 @@ export default async function prepareTask(endpoint: Endpoint, rootPath: string)
 
   if (scannerMode === ScannerMode.CLI) {
     let actualSonarOpts = tl.getVariable('SONAR_SCANNER_OPTS');
+    if (actualSonarOpts == null) {
```

Comment:
We get an error message with undefined, does this null check fixes that?

Reply:
Yes !